Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+gem "just-the-docs"
+gem "github-pages", group: :jekyll_plugins
```

Comment:
Replace this 
` gem "github-pages", group: :jekyll_plugins `
with
```
gem 'jekyll'
gem 'jekyll-feed'
gem 'jekyll-sitemap'
gem 'jekyll-seo-tag'
gem 'jemoji'
gem 'jekyll-readme-index'
```

Reply:
done